values/gitea/gitea.gotmpl

```diff
@@ -120,10 +121,14 @@ postgresql:
       memory: 256Mi
       cpu: 250m
 
-{{- if $hasCA }}
+{{- if $untrustedCA}}
 extraRootCA: |
-  {{- $v.cluster | get "customRootCA" $v.letsencryptRootCA | nindent 2 }}
-  {{- $v.cluster | get "customIntermediateCA" $v.letsencryptCA | nindent 2 }}
+{{- if eq $cm.issuer "custom-ca" }}
+  {{- ($cm | get "customRootCA" "") | nindent 2 }}
```

Comment:
```suggestion
  {{- $cm.customRootCA | nindent 2 }}
```

Reply:
that prop is mandatory when that IF statement is true

Reply:
Good point.
Schema does not guarantee that is is mandatory by at this stage we know that this property must be set

Reply:
done